trino/trino.go

```diff
@@ -1903,7 +1920,15 @@ func parseNullTimeWithLocation(v string) (NullTime, error) {
 	}
 	stamp, location := v[:idx], v[idx+1:]
 	loc, err := time.LoadLocation(location)
+	// Not a named location.
 	if err != nil {
```

Comment:
Is it possible to inspect the `err` to see if it failed to parse the value or is it an actual problem with loading TZ data? Or since we already extracted the location from the string, maybe we could check its format first?

Reply:
Instead of looking at `err` we can just look if location looks like an offset.